src/screens/HomeScreen.tsx

```
@@ -55,17 +57,27 @@
           {trips.map((trip) => (
             <li key={trip.id} className="box mb-3">
               <div className="columns">
-                <div className="column is-four-fifths">
-                  <button
-                    className="button is-link is-outlined is-fullwidth"
-                    onClick={() => navigate(`/trip/${trip.id}`)}
-                  >{trip.title}</button>
+                <div 
+                  className="column is-four-fifths is-clickable" 
+                  onClick={() => navigate(`/trip/${trip.id}`)}
+                >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)